### PR TITLE
Unary gRPC is now supported.

### PR DESCRIPTION
**Pull request by darioielardi**
https://issuetracker.google.com/issues/139258904

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -625,8 +625,8 @@ $ curl -v https://<url>
 
 ### Is gRPC supported on Cloud Run?
 
-[gRPC](https://grpc.io) is currently not supported on Cloud Run. However, [Cloud
-Run on GKE][crogke] supports running applications serving gRPC traffic.
+Cloud Run supports unary [gRPC](https://grpc.io), while streaming is not yet supported. However, [Cloud
```
Comment (ahmetb) [inline README.md]: the GKE part is actually not true.
it "works" but it's not actively "supported".

**Comment by ahmetb** [comment]
Thanks for the update, happy to merge it if you can address the comments.

**Comment by darioielardi** [comment]
Fair enough, updated.

**Comment by darioielardi** [comment]
What's the problem here? Something I should fix?

**Pull request merged**